src/mrinufft/trajectories/io.py

```diff
@@ -245,13 +264,13 @@ def read_trajectory(
     grad_filename : str
         Gradient filename.
     dwell_time : float, optional
-        Dwell time of ADC, by default 0.01
+        Dwell time of ADC, by default 0.1
```

Comment:
Missing unit

Reply:
Done, and the value was changed back as this one was a Ctrl-H mistake.